services/ui-src/src/routes/ReadOnly/ReadOnly.tsx

```diff
@@ -0,0 +1,58 @@
+import { useNavigate } from "react-router-dom";
+// components
+import { Box, Button, Flex, Heading } from "@chakra-ui/react";
+import { Form } from "components";
+// types
+import { AnyObject } from "types";
+// form
+import formJson from "forms/internal/adminDashSelector/adminDashSelector";
+import formSchema from "forms/internal/adminDashSelector/adminDashSelector.schema";
+
+export const ReadOnly = ({ verbiage }: Props) => {
+  const navigate = useNavigate();
+
+  const onSubmit = (formData: AnyObject) => {
+    const selectedState = formData["ads-state"];
+    localStorage.setItem("selectedState", selectedState);
+    navigate("/mcpar/dashboard");
```

Comment:
`yarn coverage` shows these lines needing test coverage. Can probably add this as part of your current submission tests. As part of this, you might have to mock localStorage, which can be a bit tricky. These two StackOverflow answers should get you headed the right direction, but I'm happy to pair too.
- https://stackoverflow.com/a/54157998/8908984
- https://stackoverflow.com/a/47897345/8908984

Reply:
Looks like we don't need this 💯